source/common/network/address_impl.cc

```diff
@@ -50,7 +50,8 @@ bool ipFamilySupported(int domain) {
   Api::OsSysCalls& os_sys_calls = Api::OsSysCallsSingleton::get();
   const Api::SysCallIntResult result = os_sys_calls.socket(domain, SOCK_STREAM, 0);
   if (result.rc_ >= 0) {
-    RELEASE_ASSERT(os_sys_calls.close(result.rc_).rc_ == 0, "");
+    RELEASE_ASSERT(os_sys_calls.close(result.rc_).rc_ == 0,
+                   absl::StrCat("fail to close fd ", result.rc_));
```

Comment:
Thanks for adding details here.

maybe -> "failed to close fd: response code " ?
since right now it looks like you're printing the fd, not the error code.